src/Symfony/Component/Ldap/Security/LdapUserProvider.php

```diff
@@ -81,7 +81,7 @@ public function loadUserByIdentifier(string $identifier): UserInterface
         }
 
         $identifier = $this->ldap->escape($identifier, '', LdapInterface::ESCAPE_FILTER);
-        $query = str_replace(['{username}', '{user_identifier}'], $identifier, $this->defaultSearch);
+        $query = str_replace('{user_identifier}', $identifier, $this->defaultSearch);
```

Comment:
This line is not directly depended on `LdapBadge`'s values. Let's be on the safe side and add a `{username}` check + deprecation here as well (like you added to the badge).